src/languages/__tests__/cs.test.js

```diff
@@ -0,0 +1,7 @@
+import { config } from '../cs';
+
+describe('config', () => {
+  it('should return localization object', () => {
+    expect(config({ companyNames: ['test1', 'test2', 'test3'] })).toMatchSnapshot();
+  });
+});
```

Comment:
Proof of concept for table testing, so that all configs are checked using one test to DRY:

```suggestion
import { config as configCs } from '../cs';
import { config as configDe } from '../de';

describe.each([
  ['cs', configCs],
  ['de', configDe]
])('%s config', (language, config) => {
  it('should return localization object', () => {
    expect(config({companyNames: ['test1', 'test2', 'test3']})).toMatchSnapshot();
  });
})
```

Reply:
Refactored.